### PR TITLE
Added Student Version support

### PR DESCRIPTION
**Pull request by maxcapodi78**
Bug fix in Edb.py Database Opening

### CODE REVIEW

**Comment by akaszynski** [comment]
We need to document the `student_version` parameter before merging this in the applicable classes.

File: pyaedt/generic/general_methods.py [hunk 1]
```diff
@@ -201,6 +201,55 @@ def env_value(input_version):
...
+            version -= 1
+        else:
+            release -= 2
+    v_key = "ANSYSEMSV_ROOT{0}{1}".format(version, release)
+    return os.getenv(v_key)
+
+
+@aedt_exception_handler
+def env_value_student(input_version):
+    """
```
Comment (akaszynski) [inline pyaedt/generic/general_methods.py:231]: Would be nice to have this documented (at least a provide a summary statement).

File: pyaedt/generic/general_methods.py [hunk 2]
```diff
@@ -201,6 +201,55 @@ def env_value(input_version):
     v_key = "ANSYSEM_ROOT{0}{1}".format(version, release)
     return v_key
 
+
+@aedt_exception_handler
+def env_path_student(input_version):
+    """
```
Comment (akaszynski) [inline pyaedt/generic/general_methods.py:207]: Would be nice to have this documented (at least the short summary).

File: pyaedt/edb.py [hunk 3]
```diff
@@ -177,31 +181,23 @@ def open_edb(self, init_dlls=False):
...
-                self.builder = self.layout_methods.GetBuilder(self._db, self._active_cell)
-            else:
-                print("opening from EDBLib")
-                self.builder = self.layout_methods.OpenEdbStandAlone(self.edbpath, self.edbversion)
-                self._db = self.builder.EdbHandler.dB
-                self._active_cell = self.builder.EdbHandler.cell
+        print(self.edbpath)
+        print(self.edbversion)
+        print(_ironpython)
+        print(dir())
```
Comment (akaszynski) [inline pyaedt/edb.py:187]: We should avoid using print statements and instead use the `logging` module.  Many users will want to suppress output to stdout or will want to have a different level of logging depending on their workflow.  Using the `logging` module lets them do that.

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.  `print` --> `logging` can be saved for a different PR since it's out of scope of this PR, but we really should implement that.

File: pyaedt/generic/general_methods.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/generic/general_methods.py]: Done

File: pyaedt/edb.py (hunk 3 above)
Comment (maxcapodi78) [reply pyaedt/edb.py]: Done. I've replaced with messenger calls

**Pull request merged**